addons/mail/static/src/models/messaging_notification_handler/messaging_notification_handler.js

```diff
@@ -682,6 +684,23 @@ function factory(dependencies) {
             });
         }
 
+        /**
+         * @private
+         * @param {integer} channelId
+         * @param {Object} messageData
```

Comment:
"message" is confusing in this context, should specify "busMessage" or "payload"